README.Rmd

```diff
@@ -219,7 +219,7 @@ nowcast <- epinowcast(data = pobs,
 
 ### The `epinowcast` object
 
-The `epinowcast()` function returns an `epinowcast` object which includes diagnostic information, the data used for fitting, and the underlying `cmdstanr` object.
+The `epinowcast()` function returns an `epinowcast` object which includes diagnostic information, the data used for fitting, and the underlying `cmdstan` object.
```

Comment:
I think cmdstanr calls this CmdStanModel https://mc-stan.org/cmdstanr/reference/CmdStanModel.html

Reply:
Good shout. I don't think we are doing this correctly in multiple places. Will make an issue.